src/pet.js

```diff
@@ -0,0 +1,73 @@
+const MIN_FITNESS = 0;
+const MAX_FITNESS = 10;
+const MIN_HUNGER = 0;
+const MAX_HUNGER = 10;
+const MAX_AGE = 30;
+
+function Pet(name) {
+    this.name = name
+    this.age = 0
+    this.hunger = 0
+    this.fitness = 10
+    this.children = []
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < MAX_AGE && this.hunger < MAX_HUNGER && this.fitness > MIN_FITNESS;
```

Comment:
Can make this a bit cleaner to read by splitting each condition onto a new line:
```
        return this.age < MAX_AGE &&
            this.hunger < MAX_HUNGER &&
            this.fitness > MIN_FITNESS;
```